pkg/kubectl/describe.go

```diff
@@ -162,6 +162,25 @@ func init() {
 	DefaultObjectDescriber = d
 }
 
+func getStorageClassAnnotation(obj api.ObjectMeta) string {
```

Comment:
Seems to me like we should be moving `getVolumeClass` and `getClaimClass` from `controller/volume/persistentvolume/pv_controller_base.go` to `pkg/api/helpers.go`.

We should also implement an `IsDefaultStorageClass()` function in a new `pkg/apis/storage/helpers.go`

These helpers should be used everywhere. You missed the usages in `controller/volume/persistentvolume/*`